### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by zefir6** [Playabl-io/playabl#153] [General]
Are you planning to have different language options? 
Is anything that can be done to help with that (I have zero idea of front end development, but I would be willing to help and find people to help with translation).

**Comment by jongrim** [comment]
Hey @zefir6! I think it'd be pretty cool to eventually have at least portions of the site translatable in the future. The first step for that to be possible is to make some fundamental updates to the app architecture. [I have a placeholder issue to explore this in the future](https://github.com/Playabl-io/playabl/issues/127), but I'm not sure when that might be exactly.

Do you speak another language? If you'd be open to helping translate portions later I can reach back out if / when the site can support translations.

**Reply by zefir6** [reply]
I do, I speak polish, and I think I can also reach people who could do 1-2 other languages.
I also belong to SF Association in poland which organizes periodic sessions, and being able to run our community in playabl would be very cool. So ping me when you get there. You can also find me on OHG slack.
